frontend/src/components/User/GameButton.tsx

```diff
@@ -34,7 +34,15 @@ function GameButton(props: Props) {
           offline: <></>,
           online: <button onClick={inviteGame}>게임 초대</button>,
           inGame: <button onClick={watchingGame}>게임 관전</button>,
-        }[props.activity.activity]
+        }[
+          (
+            activityMap.get(props.userId) ?? {
+              userId: props.userId,
+              activity: 'offline',
+              gameId: null,
+            }
+          )?.activity
```

Comment:
아래와 같이는 할 수 없는지 확인 한번 해주세요 :)
```suggestion
            activityMap.get(props.userId)?.activity ?? 'offline'
```

Reply:
엇 아까 해봤을 때 안됐던 거 같았는데... 코드가 좀 달랐었나보네요. 해보니까 잘됩니다! 감사합니다